stringtables/COAL_localization.st

```diff
@@ -0,0 +1,130 @@
+StringTable {
+ ItemClassName "CustomStringTableItem"
+ Items {
+  CustomStringTableItem "{60F896A34D37E060}" {
+   Id "#Author_Coalition"
+   Target_en_us "Coalition"
+   Target_fr_fr "Coalition"
+   Target_it_it "Coalition"
+   Target_de_de "Coalition"
+   Target_es_es "Coalition"
+   Modified 1626905417
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0B88B7B4C}" {
+   Id "#Coal_SS_Countdown_Cancelled"
+   Target_en_us "[CRF] : Game Live Countdown Canceled!"
+   Target_de_de "[CRF] : Spielstart Countdown gestoppt!"
+   Modified 1626905676
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0A7205480}" {
+   Id "#Coal_SS_Countdown_Started"
+   Target_en_us "[CRF] : Game Live In: %1 Seconds!"
+   Target_de_de "[CRF] : Spiel startet in %1 Sekunden!"
+   Modified 1626905691
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0E6620551}" {
+   Id "#Coal_SS_Faction_Not_Ready"
+   Target_en_us "Not Ready"
+   Target_de_de "Nicht bereit"
+   Modified 1626905698
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0F2448B7A}" {
+   Id "#Coal_SS_Faction_Readied"
+   Target_en_us "[CRF] : %1 Has Readied Up a faction"
+   Target_de_de "[CRF] : %1 hat eine Fraktion für bereit erklärt"
+   Comment "Generic version of ready string in case a certain side/faction is not represented in the other strings"
+   Modified 1626905909
+   Author "pi123"
+   LastChanged "pi123"
+  }
+  CustomStringTableItem "{60F896A0C5CAEB3E}" {
+   Id "#Coal_SS_Faction_Readied_Blufor"
+   Target_en_us "[CRF] : %1 Has Readied Up Blufor"
```

Comment:
Lets change this verbiage while we're here. I'm thinking

`Side READY - name`

Example:
BLUFOR READY - tanaKa

Reply:
Sounds good, I assume for a generic version it'd just be "FACTION READY - name" then?

Reply:
Also keeping the CRF tag for ease of recognition/debugging I assume?